obspy/io/cmtsolution/__init__.py

```diff
@@ -22,19 +22,24 @@
 >>> cat = obspy.read_events("/path/to/CMTSOLUTION")
 >>> print(cat)
 1 Event(s) in Catalog:
-2003-12-26T01:56:58.129999Z | +29.100,  +58.240 | 6.54 mw
+2003-12-26T01:56:58.130000Z | +29.100,  +58.240 | 6.54 mw
```

Comment:
that's funny that new numpy makes this strange rounding-error-in-string-representation go away..

Reply:
Yea I think the doctests were actually not executed at all for a while as there was a typo in the pragma of the next doctest. I've only seen a verbose error message when using pytest's doctest runner.